wellcomeml/logger.py

```diff
@@ -27,11 +29,24 @@ def build_logger(logging_level, name):
     return logger
 
 
-LOGGING_LEVEL = os.getenv("LOGGING_LEVEL")
+DEFAULT_LOGGING_LEVEL = "INFO"
+LOGGING_LEVEL = os.getenv("LOGGING_LEVEL", DEFAULT_LOGGING_LEVEL)
+NUMERIC_LEVEL = get_numeric_level(LOGGING_LEVEL)
 
-logger = build_logger(logging_level=LOGGING_LEVEL, name=__name__)
+logger = build_logger(logging_level=NUMERIC_LEVEL, name=__name__)
```

Comment:
A bit clunky terminology. I would maybe use a union type `[str, int]` for LOGGING_LEVEL and re-use it instead of numeric_level?

Reply:
So you suggest we convert `LOGGING_LEVEL` and reassign. Sounds good 👍

Reply:
7d9ad96fdeda86b0cfe4189ebba89646b328821d